acceptance/install/config_test.go

```diff
@@ -0,0 +1,56 @@
+package acceptance_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"runtime"
+
+	"github.com/epinio/epinio/helpers/epinio"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Epinio Installation", func() {
+	var flags string
+	epinioBinary := fmt.Sprintf("../../dist/epinio-%s-%s", runtime.GOOS, runtime.GOARCH)
+
+	configFile := "./assets/tests/config.yaml"
```

Comment:
This test is altering the file which means subsequent tests will find it altered. If a similar test is added in the future, the order in which the tests are run could make this test fails. I think it's better if we copy the file to a temporary location and use it from there. The temporary file should be deleted when the tests is done.

Reply:
Done.